opm/parser/eclipse/Parser/ParseContext.hpp

```diff
@@ -198,6 +198,14 @@ class KeywordLocation;
         */
         const static std::string PARSE_MISSING_INCLUDE;
 
+        /*
+          For certain keywords, other, specific keywords are either
+          required or prohibited. When such keywords are found in an
+          invalid combination (missing required or present prohibited
+          keyword), this error situation occurs.
+         */
```

Comment:
Need to also update the `ParseContext::initDefault()` function

Reply:
The test should also set the parsecontext behavior explicitly, som that the test does not depend om the default value.

Reply:
I'm not sure what you mean here; `ParseContext::initDefault()` is updated to set the correct value (`THROW_EXCEPTION`) (see `ParseContext.cpp:84`), and what the test, well, tests, is a) that the default behaviour is throwing an exception, and b) that the exception is thrown.

Reply:
> ParseContext::initDefault() is updated to set the correct value (THROW_EXCEPTION) (see ParseContext.cpp:84

You are right - I did not see that.